src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -149,13 +171,12 @@ DlgPrefInterface::DlgPrefInterface(
             this,
             &DlgPrefInterface::slotSetScheme);
 
-    checkBoxScaleFactorAuto->hide();
-    spinBoxScaleFactor->hide();
-    labelScaleFactor->hide();
-
     // Start in fullscreen mode
-    checkBoxStartFullScreen->setChecked(m_pConfig->getValueString(
-                    ConfigKey("[Config]", "StartInFullscreen")).toInt()==1);
+    checkBoxStartFullScreen->setChecked(
+            m_pConfig
+                    ->getValueString(
+                            ConfigKey(kConfigGroup, kStartInFullscreenKey))
+                    .toInt() == 1);
```

Comment:
there are some ugly linebreaks now.
what about this, precommit agrees
```suggestion
            m_pConfig->getValueString(
                             ConfigKey(kConfigGroup, kStartInFullscreenKey))
                    .toInt() == 1);
```

Reply:
Why not make the whole config key a constant? Then the line would be shorter.